src/routes/api/installation/api-installation-syncstate-get.ts

```diff
@@ -4,9 +4,34 @@ import { Subscription } from "models/subscription";
 import { pick } from "lodash";
 
 export const ApiInstallationSyncstateGet = async (req: Request, res: Response): Promise<void> => {
+	if (req.query.limit === undefined) {
```

Comment:
could query be undefined? and need a null propagator

Reply:
No, I think it is always defined, at least according to express typing
![Uploading image.png…]()